4-Express/server/handlers/storage/getItem.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path')
+const db = require(path.resolve('server/db/','db'))
+
+const getItem = (req,res,next) => {
+    if(db[req.params.id]){
+        res.status(200).send(db[req.params.id])
+    }
+    else{
```

Comment:
most of the time you can avoid using `else` statement and not getting into this indentation/scopes hell.
by terminating the execution if the condition is met will make your code cleaner, in your case it'll be something like
```
if (!db[req.params.id]) {
    res.status(404).send('...')
    return; // this is the important change here
}

// validations has passed
res.status(200).send(...);
```

what do you think?